core/controllers/library.py

```diff
@@ -303,8 +304,7 @@ def get(self):
 
 
 class CollectionSummariesHandler(base.BaseHandler):
-    """Returns collection summaries corresponding to collection ids.
-    """
+    """Returns collection summaries corresponding to collection ids."""
```

Comment:
Here, the `"""` has been moved back to the previous line. This is not consistent with the previous behavior where it was being moved to the next line. Could you please explain?